test/over_react/util/rem_util_test.dart

```diff
@@ -369,13 +369,13 @@ main() {
           await new Future.delayed(const Duration(milliseconds: 100));
           expect(toRem('20px').number, 1);
         });
-      }, testOn: 'dartium || chrome');
+      }, testOn: 'chrome');
 
       test('only in Google Chrome', () async {
         toRem('1rem');
         await new Future.delayed(const Duration(milliseconds: 100));
         expect(querySelector('#rem_change_sensor'), isNull, reason: 'test setup sanity check');
-      }, testOn: '!chrome && !dartium');
+      }, testOn: '!chrome');
```

Comment:
Should these be `!browser`?

Reply:
Looking at the test, I'm not sure why it even had `!chrome`. Based on the description it should be `testOn: '!chrome'`

Reply:
Actually the test verifies that nothing is added to the DOM in non-chrome browsers, so `!chrome` should stay. The test name has been updated to be less confusing.